package/MDAnalysis/analysis/gnm.py

```diff
@@ -251,7 +258,10 @@ def __init__(self,
         self.u = universe
         self.select = select
         self.cutoff = cutoff
-        self.results = []  # final result
+        self.results = Results()
+        self.results.times = []
```

Comment:
Shouldn't `times` be handled in the base analysis class?

Reply:
Yes we could remove it.